### DISCUSSION TITLE
Is this the correct way to play videos of a FreeTube playlist externally?

### DISCUSSION THREAD

**Discussion by stdedos** [FreeTubeApp/FreeTube#5188] [Q&A]
Coming from https://github.com/FreeTubeApp/FreeTube/issues/1735, I see that trying to play the (current) video in `mpv` gives

```console
$ until ps faux | grep 'm[p]v'; do sleep 0.1 ; done
stdedos   389097  0.0  0.0 232212  4136 ?        Ssl  10:56   0:00  |           |   \_ flatpak-spawn --host /usr/bin/mpv --loop --audio-pitch-correction=no --start=52 --speed=1 --playlist-start=3 https://youtube.com/playlist?list=ft-playlist--6f205e0a-3e1f-4c47-b29b-2feb41594c55
stdedos   389102  0.0  0.0 242384 26996 ?        Rs   10:56   0:00  |   \_ /usr/bin/mpv --loop --audio-pitch-correction=no --start=52 --speed=1 --playlist-start=3 https://youtube.com/playlist?list=ft-playlist--6f205e0a-3e1f-4c47-b29b-2feb41594c55
$ /usr/bin/mpv --loop --audio-pitch-correction=no --start=52 --speed=1 --playlist-start=3 https://youtube.com/playlist?list=ft-playlist--6f205e0a-3e1f-4c47-b29b-2feb41594c55
[ytdl_hook] Got empty playlist, nothing to play. 
Failed to recognize file format.


Exiting... (Errors when loading file)
$
```

which is correct (nowadays https://youtube.com/playlist?list=ft-playlist--6f205e0a-3e1f-4c47-b29b-2feb41594c55 redirects back to https://youtube.com/)

Is this a bug, or a feature that I am missing?

----
_On a tangent_, it's very hard to debug issues like these. Can "some more diagnostics" come "somewhere"? :confused:
my bash-fu is high, but nothing beats "direct logs"

**Comment by stdedos** [comment]
Okay, I think this is a bug.

Playing "simply the video" works (`/usr/bin/mpv --loop --audio-pitch-correction=no --player-operation-mode=pseudo-gui --start=120 --speed=1 --playlist-start=3 https://youtube.com/playlist?list=ft-playlist--6f205e0a-3e1f-4c47-b29b-2feb41594c55
`), playing a video while a playlist is loaded doesn't (because it tries to fetch an unknown playlist)

**Reply by stdedos** [reply]
_Notice me @efb4f5ff-1298-471a-8973-3d47447115dc_ 🙏

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
I dont know the answer to this. Please do not ping members of the team just to force a response. We or the community will eventually get back to you.

**Reply by stdedos** [reply]
I have seen you noticing issues pretty quickly, yes. I didn't know if you are noticing discussions too.

**Comment by absidue** [comment]
I've confirmed it is a bug, when you are watching a user playlist, FreeTube should only pass the video information to the external player, but the check for that was incorrect on the watch page.
I've opened this pull request to fix the bug: https://github.com/FreeTubeApp/FreeTube/pull/5263
